templates/web/base/report/form/user_loggedout.html

```diff
@@ -0,0 +1,47 @@
+[% IF c.cobrand.social_auth_enabled %]
+  [% IF c.config.FACEBOOK_APP_ID %]
+    <div class="form-box">
+    <button name="facebook_sign_in" id="facebook_sign_in" value="facebook_sign_in" class="btn btn--block btn--social btn--facebook">
+        <img alt="" src="/i/facebook-icon-32.png" width="17" height="32">
+        [% loc('Log in with Facebook') %]
+    </button>
+    </div>
+  [% END %]
+  [% IF c.config.TWITTER_KEY %]
+    <div class="form-box">
+    <button name="twitter_sign_in" id="twitter_sign_in" value="twitter_sign_in" class="btn btn--block btn--social btn--twitter">
+        <img alt="" src="/i/twitter-icon-32.png" width="17" height="32">
+        [% loc('Log in with Twitter') %]
+    </button>
+    </div>
+  [% END %]
+	<div class="form-box hidden-nojs">
+		<button class="btn btn--social btn--social-email">[% loc('Log in with email') %]</button>
```

Comment:
This doesn't cause the validation to run but the FB and twitter buttons do.

Reply:
Which validation do you mean here? The login with email won't because it just shows a new form that will then validate later, whereas the social ones will be sending you offsite immediately so do it then, if that makes sense?